src/makeCalculator.js

```diff
@@ -4,7 +4,54 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    get result() {
+      return result;
+    },
```

Comment:
[CODE KNOWLEDGE] - The getter for `result` is unnecessary since the `result` property should be directly accessible. Instead of using a getter, you can simply have `result: 0` in the object and update it within the methods.